server/controllers/recipes.js

```diff
@@ -323,6 +324,8 @@ export class Recipes {
    */
   getRecipeById(req, res) {
     const { recipeId } = req.params;
+    const { currentUser } = req;
+
```

Comment:
These are the only place you're using `req`, therefore you can destructure the params.
```
getRecipeById({params: {recipeId}, currentUser}, res) {
```